pages/new-tab/src/NewTab.tsx

```diff
@@ -3,12 +3,15 @@ import '@src/NewTab.scss';
 import { useStorageSuspense, withErrorBoundary, withSuspense } from '@extension/shared';
 import { exampleThemeStorage } from '@extension/storage';
 import { Button } from '@extension/ui';
+import { t } from '@extension/i18n';
 
 const NewTab = () => {
   const theme = useStorageSuspense(exampleThemeStorage);
   const isLight = theme === 'light';
   const logo = isLight ? 'new-tab/logo_horizontal.svg' : 'new-tab/logo_horizontal_dark.svg';
 
+  console.log(t('hello', 'World'));
```

Comment:
OO there's some example, but as i type above, this should be also in README 😸

Reply:
I thought I documented it well enough in the README.md in the i18n package, but what do you think?

Reply:
I think, if you implement 'placeholder' filed in json, there's no need to create more clarification, but no it's a little tricky to use it 😆